code/modules/clothing/head/jobs.dm

```diff
@@ -18,6 +18,7 @@
 		slot_r_hand_str = "caphat",
 		)
 	body_parts_covered = 0
+	style = STYLE_HATHATHAT
```

Comment:
Here and in other items STYLE_HATHATHAT seems like an overkill, STYLE_HIGH will be more than enough.

Reply:
hats are stylish in all cases and so they have an intrinsic +1 style minimum and +2 style average.